docs/install.md

```diff
@@ -2,6 +2,23 @@
 
 Use this page to add the component to an existing Kubernetes cluster.
 
+## Pre-requisites
+
+1. A Kubernetes cluster (_if you don't have an existing cluster_):
+```bash
+   # Example cluster creation command on GKE
+   gcloud container clusters create $CLUSTER_NAME \
+     --zone=$CLUSTER_ZONE
+   ```
+
+2. Grant cluster-admin permissions to the current user:
+   ```bash
```

Comment:
Indentation is off by a tab. Markdown render looks a bit weird

Reply:
Done